Update text_ui.py

atummons opened the pull request:
Reverts back to where it used to be.  This change throws an error (maybe an issue upstream) where various events players will error out.  This makes Text_UI have the issue from before where it won't update player variables unless score, ball, or player changes.  But better than breaking.  At some point I will dig into a real fix to make Text_UI match what is expected for all scenarios.

atummons committed:
Update text_ui.py

Reverts back to where it used to be.  This change throws an error (maybe an issue upstream) where various events players will error out.  This makes Text_UI have the issue from before where it won't update player variables unless score, ball, or player changes.  But better than breaking.  At some point I will dig into a real fix to make Text_UI match what is expected for all scenarios.

jabdoa2 commented:
I guess a try catch inside the handler would work around all those issues with non trivial types.

atummons committed:
Update text_ui.py

Per Jan comment, adds try and excepts ValueError (only error I could hit).  I just have it passing for now.  Could add specific messages to debug log, but its not really an issue, just based on how it is all designed from my view.

atummons commented:
> I guess a try catch inside the handler would work around all those issues with non trivial types.

Good idea.  Put it in, and is working for my local tests.

jabdoa2 commented:
Awesome. Thanks :-)
